specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json

```diff
@@ -152,6 +152,52 @@
                     "application/json"
                 ]
             }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.InfrastructureInsights.Admin/regionHealths/{location}/alerts/{alertName}/remediate": {
```

Comment:
Please don't call the post action remediate and then describe it as repair.  This does not repair an alert.

Reply:
As the powershell cmdlet and UI will be using repair I aligned the text in this file with that.

Reply:
The api action need not match what you need in powershell.. in any case Repair does not sound like the right term here